api/src/main/java/jakarta/resource/ResourceException.java

```diff
@@ -120,6 +120,7 @@ public String getErrorCode() {
      * <code>initCause</code> methods of the 
      * <code>java.lang.Throwable</code> class..
      */
+    @Deprecated
```

Comment:
When deprecating, you should tell the user what you expect them to use instead.

Reply:
I do not deprecate anything, see `@javadoc` in javadoc. What do you want the person who just follows compiler advice to add there?

Reply:
I noticed that the [Compatibility Requirements](https://eclipse-ee4j.github.io/jakartaee-platform/CompatibilityRequirements) which I was reading for a different review comment state that "the @Deprecated annotation must not be used".

Reply:
reverted the change